class-tgm-plugin-activation.php

```diff
@@ -3406,7 +3459,12 @@ public function after_flush_output() {
 	class TGM_Utils {
 
 		/**
-		 * @var bool $has_filters Whether the PHP filter extension is enabled
+		 * Whether the PHP filter extension is enabled.
+		 *
+		 * @var bool $has_filters True is the extension is enabled.
+		 *
+		 * @see http://php.net/book.filter
```

Comment:
Like `@param` and `@return` come last, I think `@var` should come last too. The `@see` should come immediately after the description.

Also `@since` missing.